src/platform_impl/web/script/worker.min.js

```diff
@@ -0,0 +1 @@
+onmessage=e=>{let[s,a]=e.data,l=()=>{s.postMessage(void 0)};"scheduler"in globalThis?globalThis.scheduler.postTask(l,{delay:a}):setTimeout(l,a)};
```

Comment:
new line at the end of file.

Reply:
Haha, that's the whole point!
Its supposed to save bytes as much as possible, even newlines.

Anyway, this is tool generated.

Reply:
could you indicate that it's generated?

Reply:
How would you want me to indicate that?
The indication should be the extension: `.min.js`, it means its minimized, nobody does that by hand.

Reply:
ah, so the convention is to name generated with `min`? Then it's fine. I was thinking about `.generated.js`.